src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -196,4 +210,24 @@ private String decryptResponse(String responseBody) throws ParseException, IOExc
         return isEncrypted ? hyperwalletEncryption.decrypt(responseBody) : responseBody;
     }
 
+    public WebResource getWebResource(final String url) {
+        client.addFilter(new HTTPBasicAuthFilter(this.username, this.password));
+        webResource = client.resource(url);
+        return webResource;
```

Comment:
Can be changed to: return client.resource(url);

Reply:
Changed as suggested.